keras_nlp/models/gpt_neo_x/gpt_neo_x_preprocessor_test.py

```diff
@@ -113,19 +113,18 @@ def test_serialization(self):
         )
 
     @pytest.mark.large
-    @pytest.mark.tf_only
     def test_saved_model(self):
```

Comment:
We should leave this test as is. This should be marked `tf_only`, and we should test `self.preprocessor`, not `self.tokenizer` as before.